partials/home-content.php

```diff
@@ -4,7 +4,9 @@
 
 if ($home_content) {
   foreach ($home_content as $item) {
-    echo '<div class="home-item u-pointer">';
+    echo '<div class="home-item margin-top-';
+    echo rand(0,1) == 1 ? 'mid' : 'basic';
+    echo '"><div class="home-item-holder">';
```

Comment:
the only difference is the holder has inline block. which i dont think is needed